modules/base/manifests/mounts.pp

```diff
@@ -104,18 +102,33 @@
         group   => 'govuk-backup',
     }
 
-    lvm::volume { 'graphite':
-        ensure  => present,
-        pv      => '/dev/sde',
-        vg      => 'graphitebackup',
-        fstype  => 'ext4',
+    # lvm::volume { 'graphite':
```

Comment:
Yes, it follows convention. See the above example. It's also good because it points out that this should have been using the `lvm::volume` resource, but we had to unpack that to make use of arrays.